guide/creating-your-bot/adding-more-commands.md

```diff
@@ -59,7 +59,7 @@ if (message.content.startsWith(`${prefix}ping`)) {
 Now the ping command will trigger whenever the message _starts with_ `!ping`! Sometimes this is what you want, but other times, you may want to match only exactly `!ping` - it varies from case to case, so be mindful of what you need when creating commands.
 
 ::: warning
-Be aware that this will also match `!pingpong`, `!pinguin`, and the like. This is not a huge problem for now, so don't worry; you'll see better ways to check for commands later.
+Be aware that this will also match `!pingpong`, `!pinguin`, and the like. This is not a huge problem, for now, so don't worry; you'll see better ways to check for commands later.
```

Comment:
```suggestion
Be aware that this will also match `!pingpong`, `!pinguin`, and the like. This may or may not be a problem for you, but don't worry; you'll see better ways to check for commands later.
```